src/pools/abstract-pool.ts

```diff
@@ -2,6 +2,13 @@ import EventEmitter from 'events'
 import type { MessageValue } from '../utility-types'
 import type { IPool } from './pool'
 
+/**
+ * An intentional empty function
```

Comment:
JSDoc wants a satisfied description. So it needs to end with a period.

```suggestion
 * An intentional empty function.
```